apps/src/sites/code.org/pages/learn/index.js

```diff
@@ -0,0 +1 @@
+import '@cdo/apps/tutorialExplorer/tutorialExplorer';
```

Comment:
I think tutorialExplorer might benefit from the same pattern we used of getting data from dashboard (or in this case pegasus) through a data attribute on the script tag.